src/app/pages/tags/list/tags-list-table-data-source.ts

```diff
@@ -146,8 +147,10 @@ export class TagsListTableDataSource extends TableDataSource<Tag> {
         this.exportAction.visible = tags.canExport;
         this.deleteManyAction.visible = tags.canDelete;
         this.unassignManyAction.visible = tags.canUnassign;
+        this.unassignManyAction.visible = tags.canUnassign;
```

Comment:
Same line in 149 (unassignManyAction).
Could you check?

Reply:
Added new line by error. 
Fixed.